db/migrate/20181114165757_add_utc_time_to_measurement.rb

```diff
@@ -0,0 +1,5 @@
+class AddUtcTimeToMeasurement < ActiveRecord::Migration
+  def change
+    add_column :measurements, :arrival_utc_time, :datetime
```

Comment:
It's interesting that thinking about the name brought us to something similar to what is formalized https://en.wikipedia.org/wiki/Time_of_arrival

Reply:
See for yourself if that formalization is a better naming. In any case, I feel what is in place now is already a good one